spec/factories/attachments.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :attachment do
+    skip_create
+    
+    sequence(:filename) { |n| "image#{n}.png" }
+    node { Node.issue_library }
```

Comment:
I think that just adding `node` will let FactoryGirl figure out the rest, we don't need it to be the `.issue_library`